transform/transformers/PDFTransformer.py

```diff
@@ -28,22 +30,32 @@
 styles = getSampleStyleSheet()
 
 # Basic text style
-styleN = styles["BodyText"]
-styleN.alignment = TA_LEFT
-styleN.spaceAfter = 25
+style_n = styles["BodyText"]
+style_n.alignment = TA_LEFT
+style_n.spaceAfter = 25
 
 # Subheading style
-styleSH = styles["Heading2"]
-styleSH.alignment = TA_LEFT
+style_sh = styles["Heading2"]
+style_sh.alignment = TA_LEFT
 
 # Sub-subheading style (questions)
-styleSSH = styles["Heading3"]
-styleSSH.alignment = TA_LEFT
+style_ssh = styles["Heading3"]
+style_ssh.alignment = TA_LEFT
 
 # Main heading style
 styleH = styles['Heading1']
 styleH.alignment = TA_CENTER
 
+
+# Answer Style
+style_answer = copy(styles["BodyText"])
```

Comment:
Do you definitely want a shallow copy here? What's the underlying data structure of `styles["BodyText"]`?